index.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html lang="de">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link rel="stylesheet" href="css/style.css"
+  
+  
+</head>
+
+<body>
+  <header>
+    <img src="pictures/Gorillas.jpg" alt="logo">
+  <div class="topnav" id="myTopnav">
+      <a href="#index.html" class="active">Home</a>
+      <a href="info.html">Infoseite</a>
+      <a href="kontakt.html">Kontakt</a>
+      <a href="#about">About</a>
+      <a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776;</a>
+    </div>
+    <h1>Unser Zoo und seine Bewohner</h1>
+    <hr>
+  </header>
+  
+  <main>
+    <article>
+      <h1>Komm in den Zoo Schönbrunn</h1>
+      <h2>Es erwarten dich viele Tiere, z.B. unser Nachwuchs: <br>ein Eisbärenbaby</h2>
+      <p>Das ist einfach ein Text. Das ist einfach ein Text. Das ist einfach ein Text.<br>
+      
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/ZK4AHFZjbTM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <br>Erlebe einen schönen Tag bei uns.
+          <h3>Unsere Tierwelt nach Arten </h3>
+      <dl>
+        <dt>Säugetiere, wie</z></dt>
+            <dd>Wildpferde</dd>
+            <dd>Raubkatzen</dd>
+            <dd>Elefanten</dd>
+        <dt>Fische, wie</dt>
+            <dd>Haie</dd>
+            <dd>Barsche</dd>
+        <dt>Insekten, wie</z></dt>
+            <dd>Ameisen</dd>
+            <dd>Gottesanbeterinnen</dd>
+            <dd>Elefanten</dd>
+        <dt>Vögel, wie</z></dt>
+            <dd>Adler</dd>
+            <dd>Papageien</dd>
+            <dd>Flamingos</dd>
+
+
+        
+<br>
+
+        <blockquote cite=“http://zitate.net/tier-zitate“><p> Die Tiere empfinden wie der Mensch Freude und Schmerz, Glück und Unglück </p></blockquote>

```

Comment:
die hochkommata sind nicht die richtigen (aus einem textverarbeitungsprogramm/webseite/... raus kopiert??, solche Hochkommata können in keinen editor geschrieben werden...)